src/OSPSuite.Infrastructure.Import/Services/PKParameterSensitivitiesImportFile.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace OSPSuite.Infrastructure.Import.Services
+{
+   public class PKParameterSensitivitiesImportFile : ImportLogger
+   {
+      
+   }
+}
```

Comment:
? What is this class for?

Reply:
It is used just to be consistent with all other files import where we have a dedicated class. I can remove it if you prefer.